src/dist/conf/application.conf

```diff
@@ -14,13 +14,18 @@ service {
   # Service name
   name = ${SERVICE_NAME}
 
-  # Hostname. If not supplied, taking hostname from jolokiaUrl
+  # Hostname. If not supplied, taking hostname from jolokiaFullUrl
   host = ${?SERVICE_HOST}
+
+  poller {
```

Comment:
something is weird here. This is duplicate with metricsClient above but missing params?